Review of the Getting Started content (#404)
neo4j/docs-getting-started: modules/ROOT/pages/reference/resources.adoc

[[getting-started-resources]]
= Getting started resources
:tags: get-started, introduction, resources, neo4j-help, cypher, graph, nosql-graph, neo4j-training
:page-pagination:
:!page-layout:
:page-hide-nav-title: true
:page-toclevels: 3

> To help you along your path of learning more about Neo4j, we want to provide you with the resources which may encourage you to dive deeper into Neo4j products and graph technology.

Reviewer: ```suggestion
To help you along your learning path about Neo4j, the following are some resources that may encourage you to dive deeper into Neo4j products and graph technology.
```

== Graph database concepts
* https://www.youtube.com/playlist?list=PL9Hl4pk2FsvWn1M0HMOta_9YpN930Ai8R[Video Series: Under the Hood]
* https://www.youtube.com/watch?v=5Tl8WcaqZoc&list=PL9Hl4pk2FsvWM9GWaguRhlCQ-pa-ERd4U[Video Series: Intro to Graph Databases^]
* https://neo4j.com/graph-databases-book/[Free eBook: O'Reilly Graph Databases^]
* https://dzone.com/articles/graph-databases-for-beginners-native-vs-non-native[DZone: Graph Databases for Beginners^]

[#rdbms-graph-resources]
== Graph for relational developers
* https://neo4j.com/resources/rdbms-developer-graph-white-paper/[Free eBook: Relational to Graph^]
* https://dzone.com/refcardz/from-relational-to-graph-a-developers-guide[DZone Refcard: From Relational to Graph^]
* https://neo4j.com/developer/data-modeling/[Relational to Graph Data Modeling^]
* https://medium.com/neo4j/tap-into-hidden-connections-translating-your-relational-data-to-graph-d3a2591d4026[Neo4j ETL Tool^]
* https://neo4j.com/developer-blog/neo4j-devtools-happy-new-year-new-data-importer-and-neo4j-browser-updates/[New Data Importer and Neo4j Browser Updates^]

[#cypher-resources]
== Cypher query language

=== Cypher basics and documentation

> * https://graphacademy.neo4j.com/courses/cypher-fundamentals/[GraphAcademy: Cypher Fundamentals]
> * https://neo4j.com/docs/cypher-manual/current/[Cypher documentation]
> * https://neo4j.com/docs/cypher-cheat-sheet/current/[Cypher Cheat Sheet]

Reviewer: ```suggestion
* https://neo4j.com/docs/cypher-manual/current/[Cypher documentation]
* https://neo4j.com/docs/cypher-cheat-sheet/current/[Cypher Cheat Sheet]
* https://graphacademy.neo4j.com/courses/cypher-fundamentals/[GraphAcademy: Cypher Fundamentals]
```

* https://community.neo4j.com/c/neo4j-graph-platform/cypher[Neo4j Community forum]
* https://neo4j.com/product/cypher-graph-query-language/[Cypher Query Language^]
* xref:tutorials/cypher-sql/[From SQL to Cypher]

[#cypher-sql]
=== Cypher for SQL developers
* https://youtu.be/NO3C-CWykkY[Video: SQL to Cypher^]
* https://neo4j.com/whitepapers/rdbms-developers-graph-databases-ebook/[Free eBook: Graphs for RDBMS Developers^]

[#nosql-graph-resources]
=== Graph for NoSQL developers
* https://dzone.com/articles/nosql-database-types-1[DZone: NoSQL Database Types^]

[#other-cypher-resources]
=== Other resources
* https://medium.com/neo4j/cypher-query-optimisations-fe0539ce2e5c[Medium Blog: Cypher Optimization^]
* Blog series, Handling Dates and Temporals in Cypher: https://neo4j.com/developer-blog/cypher-sleuthing-dealing-with-dates-part-1/[Part 1^], https://neo4j.com/developer-blog/cypher-sleuthing-dealing-with-dates-part-2/[Part 2^], https://neo4j.com/developer-blog/cypher-sleuthing-dealing-with-dates-part-3/[Part 3^]
* https://markhneedham.com/blog/tag/cypher/[Blog: Mark Needham on Cypher^]

> * https://maxdemarzi.com/category/cypher/[Blog: Max De Marzi on Cypher^]

Reviewer: How accurate and up-to-date are these? Neither Mark nor Max have been Neo employees for years.

Author: I didn't review this page yet, just added a few resources. Sorry, the PR is a bit confusing about what pages I actually reviewed the content. I only worked on the content of "what is neo4j", what is cypher and what is a graph database, so please don't worry so much about the rest

* link:https://graphacademy.neo4j.com/categories/beginners[Neo4j Courses for Developers^]
** link:https://graphacademy.neo4j.com/courses/app-go/[Building Neo4j Applications with Go^]
Learn how to interact with Neo4j from your Go application using the Neo4j Go Driver
** link:https://graphacademy.neo4j.com/courses/app-nodejs/[Building Neo4j Applications with Node.js^]
Learn how to interact with Neo4j from Node.js using the Neo4j JavaScript Driver
** link:https://graphacademy.neo4j.com/courses/app-python/[Building Neo4j Applications with Python^]
Learn how to interact with Neo4j from Python using the Neo4j Python Driver

* link:https://graphacademy.neo4j.com/categories/certification[Neo4j Certifications^]
** link:https://graphacademy.neo4j.com/courses/neo4j-certification/[Neo4j 4.0 Certified Professional^] 
** link:https://graphacademy.neo4j.com/courses/gds-certification/[Neo4j Graph Data Science Certification^] 

Classroom and virtual training classes, webinars:

* link:https://neo4j.com/events/world/training/[Register for public virtual and classroom training^]
* link:https://neo4j.com/webinars/[Neo4j webinars: upcoming live and on-demand^]

> Companies can also choose the instuctor-led training courses and mailto:training@neo4j.com[request] private, custom training events for internal staff to solve specific problems or understand concepts and architecture specific to their use case.

Reviewer: Unsure if this is something we should mention at all?
